### ISSUE TITLE
Create canned issue responses

### ISSUE THREAD

**Issue by brian-brazil** [prometheus/prometheus#4196]
There's a few common-ish classes of issues that it'd be nice to have canned responses for, and are general enough to do so:

- This is a support question best asked on -users
- Your idea is good, but doesn't make sense within Prometheus
- We're not accepting new SD mechanisms currently

**Comment by caniszczyk** [comment]
I have some thoughts here, we can try to use a bot to help:

https://probot.github.io/apps/welcome/
https://probot.github.io/apps/support/

All ears on other ideas.

**Comment by krasi-georgiev** [comment]
@caniszczyk  +1 for the support bot.

The vscodebot also has an interesting feature to suggest possible duplicates.
https://github.com/Microsoft/vscode/issues/50466

**Comment by RichiH** [comment]
Didn't have time to look at the bots yet, but unless they do more than just c&p a snippet, we should avoid those. We have a blurb in the issue & PR templates and if people ignore that one, they will ignore a bot as well.

The initial response should come from a human; a (dumb) bot will never be friendly, or welcoming.

**Comment by discordianfish** [comment]
:+1: :100: I'm really frustrated by Kubernetes' feja bot auto-closing issues and stuff. Nothing says "our time is worth more than yours" than such bot.
A bot is fine but it should only do stuff it was explicitly asked by a human, so might make sense for triaging/tagging etc.

**Comment by RichiH** [comment]
https://github.com/prometheus/docs/pull/1051

**Comment by brian-brazil** [comment]
I don't think we can close this jet yet, there's still outstanding comments from multiple people on that PR with no consensus.

**Comment by krasi-georgiev** [comment]
I am 50/50 for auto closing. Yes it is a bit rude , but also serves as a reminder to everyone in the issue to revisit.

The auto closing can be after 90 days and some friendly message

**Comment by sonali-rajput** [comment]
is this still getting worked on ?

**Comment by RichiH** [comment]
Not really. Are you interested in helping?

**Comment by sonali-rajput** [comment]
> Not really. Are you interested in helping?

Thank you! but I am not familiar with this at all , I am scared to make it worse and there is already some PR related to this issue right?

**Comment by kaffarell** [comment]
I think something like this would be useful. (We have 500+ issues open, many obsolete)

**Comment by krajorama** [comment]
Hello from the bug scrub! Unified canned responses have not been asked for for a long time , we closing this issue. Please follow up if you want to work on this.
